### PR TITLE
Improve `dmsg` logging

### LINKED ISSUE #10: Improve `dmsg` logging
Moved from https://github.com/skycoin/skywire/issues/426.

Originally posted by @evanlinjin:

- [x] Receiving a frame should be under `Debug`. Only connection and disconnection, of transports and remote services, should be under `Info`.
- [x] Tests should have logging set to "error".

### PR DESCRIPTION
**Pull request by nkryuchkov**
Fixes #10 

 Changes:	
- Change logging level when receiving a frame

How to test this PR:
Run tests with `debug` logging level

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -218,18 +218,18 @@ func (c *ClientConn) Serve(ctx context.Context, accept chan<- *Transport) (err e
 				defer c.wg.Done()
 				initPK, err := c.handleRequestFrame(accept, id, p)
 				if err != nil {
-					log.WithField("remoteClient", initPK).WithError(err).Infoln("Rejected [REQUEST]")
+					log.WithField("remoteClient", initPK).WithError(err).Debugln("Rejected [REQUEST]")
```
Comment (evanlinjin) [inline client.go]: I think whether a handshake completes or not should be under `Info`. What are your thoughts?

File: client.go [hunk 2]
```diff
@@ -218,18 +218,18 @@ func (c *ClientConn) Serve(ctx context.Context, accept chan<- *Transport) (err e
...
 					if isWriteError(err) || err == ErrClientClosed {
 						err := c.Close()
 						log.WithError(err).Warn("ClosingConnection")
 					}
 					return
 				}
-				log.WithField("remoteClient", initPK).Infoln("Accepted [REQUEST]")
+				log.WithField("remoteClient", initPK).Debugln("Accepted [REQUEST]")
```
Comment (evanlinjin) [inline client.go]: Some situation here.

**Review by evanlinjin — COMMENTED** [review]
Good stuff! Please review my thoughts.

File: client.go (hunk 1 above)
Comment (nkryuchkov) [reply client.go]: @evanlinjin I agree

**Pull request merged**